### ISSUE TITLE
Add AAudio output API

### ISSUE THREAD

**Issue by romain8991** [mpv-android/mpv-android#867]
Hello everyone,

I made quite some tests and ended up with the conclusion that AAudio output API might be the best performance and quality wise audio driver on Android (while also apparently being rather easy to maintain). Since someone already worked on its implementation, I'm struggling to implement it myself in Android Studio, and wan't to share it with other users, could you please add it to the mpv port of Android? Thanks in advance, I would greatly appreciate it: https://github.com/mpv-player/mpv/pull/12261/files

Romain

**Comment by sfan5** [comment]
To reduce maintenance overhead and as a general policy mpv-android only unpatched software/libraries as released by upstream.
This means AAudio support will only show up in mpv-android once it is merged into mpv.

If you want to run it for testing you will have to merge the commits into the mpv source (`buildscripts/deps/mpv`) before compiling everything from source.

**Comment by sfan5** [comment]
of note: https://github.com/mpv-player/mpv/pull/12261#issuecomment-1962293600

> mpv-android builds with API 21, AAudio was introduced with API 26.
I don't have plans to raise the API currently, especially not for a new audio driver when audio already works perfectly fine.
